LORIS_indepth.html

```diff
@@ -581,85 +295,103 @@ <h2>DICOM archives</h2>
+            <h2>Enhanced DICOM anonymization</h2>
+            <img src="images/DICOM_Anon.png">
+          </section>
 
-				<!-- LORIS Roadmap 2016 -->
-                                <section>
-                                  <h2><font color="blue">LORIS Roadmap 2016</font></h2>
-                                  <p>
-                                  <p class="fragment roll-in">Easier Installation</p>
-                                  <p class="fragment roll-in">DQT integration and improvements</p>
-                                  <p class="fragment roll-in">API for imaging</p>
-                                  <p class="fragment roll-in">Imaging Browser Enhancements</p>
-                                  <p class="fragment roll-in">Documentation - help sections, wikis, code!</p>
-                                  <p class="fragment roll-in">Automated Testing</p>
-                                  <p class="fragment roll-in">Genomics</p>
-                                  <p class="fragment roll-in">QC modules</p>
-                                  <p class="fragment roll-in">Better stats</p>
-                                  <p class="fragment roll-in">Reduce rependencies - i.e. Quickforms</p>
-
-                                </section>
-
-		    </section>
-		    
-		    <!-- Data Querying Tool -->
-		    <section>
-		      <section>
+            </section>
+              
+<!-- DQT Column 5 -->
+        <section>      
+        <section data-background="images/DQT_Define_new.png" data-background-size="100%">	<h2 style="margin:-220px 0px 200px 0px;"><font color="12a3d8">Data Querying       Tool (DQT)<br>...it's pretty cool!</font></h2>
+               </section>
+            
+            <section>
                         <h2>Data Dictionary Builder</h2>
                         <img src="images/Data_dictionary.png">
                       </section>
+            
 		      <section>
 			<h2>Data Querying Tool</h2>
 			<img src="images/DQT_Define.png">
 		      </section>
+            
 		      <section>
-                        <h2>Data Querying Tool - <font color="yellow"> Results</font></h2>
+                <h2>Data Querying Tool - <font color="yellow"> Results</font></h2>
                         <img src="images/DQT_Results.png">
                       </section>
+            
 		      <section>
                         <h2>Data Querying Tool - <font color="yellow">Statistics</font></h2>
                         <img src="images/DQT_Stats.png">
                       </section>
 		    </section>
-
-		    <section>
-		     <section>
+    
+<!-- More Cool Features Column 6-->
+              
+      <section>        
+         <section data-background="images/cat_cool.jpg">
+             <h2><font color="f27c30">LORIS has some other<br><font color="12a3d8">cool</font> features...</font></h2>
+              </section>  
+          
+          <section>
 		        <h2>Mobile <font color="yellow"> - Responsive layout</font></h2>
```

Comment:
yellow -> orange, to match the loris colour scheme, if you can

Reply:
done